docs/linux.md

```diff
@@ -80,24 +149,57 @@ Choose **oracle-oci-datasource** as your data source type.
 You can then choose between two different modes as **Tenancy mode**:
 
 * **single**: to use a single specific Tenancy
-* **multitenancy**: to use all the tenancies found in the OCI config file.
+* **multitenancy**: to use multiple tenancies
+
+### Configure Plugin in Single Tenancy Mode
+If you selected **single** as **Tenancy mode** then fill in the following credentials:
 
-If you selected **single** as **Tenancy mode** then fill in your **Tenancy OCID** and **Default Region**.
+* `Profile Name` - A user-defined name for this profile. In **single** mode this is automatically set to **DEFAULT** and cannot be modified.
+* `Region` - An OCI region. To get the value, see [**Getting Region Configuration value**](#getting-the-region).
+* `User OCID` - OCID of the user calling the API. To get the value, see [**Getting User OCID Configuration value**](#getting-the-user-OCID).* 
+* `Tenancy OCID` - OCID of your tenancy. To get the value, see [**Getting Tenancy OCID Configuration value**](#getting-the-tenancy-OCID).
+* `Fingerprint` - Fingerprint for the key pair being used. To get the value, see [**Getting Fingerprint Configuration value**](#getting-the-api-key-fingerprint).
+* `Private Key` - The contents of the private key file. To get the value, see [**Getting Private Key Configuration value**](#getting-the-private-key).
 
-![Screen Shot 2018-12-17 at 3.25.33 PM](images/Screenshot_20221206_094120.png)
+The configured data source will look like the following:
 
+![Datasource Filled](images/datasource_conf_filled.png)
 
+Click **Save & Test** to return to the home dashboard.
+
+
+### Configure Plugin in Multi-Tenancy Mode
+If you selected **multi** as **Tenancy mode** then fill in the following credentials for **each Tenancy you want to configure (up to 5 additional tenancies)**:
+
+* `Profile Name` - A user-defined name for this profile. The first Tenancy is automatically set to **DEFAULT** and cannot be modified. You need to specify a custom and unique Profile name for each of the additional tenancies.
+* `Region` - An OCI region. To get the value, see [**Getting Region Configuration value**](#getting-the-region).
+* `User OCID` - OCID of the user calling the API. To get the value, see [**Getting User OCID Configuration value**](#getting-the-user-OCID).
+* `Tenancy OCID` - OCID of your tenancy. To get the value, see [**Getting Tenancy OCID Configuration value**](#getting-the-tenancy-OCID).
+* `Fingerprint` - Fingerprint for the key pair being used. To get the value, see [**Getting Fingerprint Configuration value**](#getting-the-api-key-fingerprint).
+* `Private Key` - The contents of the private key file. To get the value, see [**Getting Private Key Configuration value**](#getting-the-private-key).
+
+By default, if you selected **multi** as **Tenancy mode** you can configure one DEFAULT tenancy and an additional one. You may add others tenancy **(up to 5 additional tenancies)** using the **Add another Tenancy** checkbox.
```

Comment:
and an additional one sounds like just one more. May want to rephrase

Reply:
It is exactly how it works. In multitenancy mode you can configure the DEFAULT and a one more. If you need more tenancies then you wil lhave to check the box and add the additionals.